loss-detection-app/res/device-mqtt/devices/device-list.yaml

```diff
@@ -0,0 +1,42 @@
+# Copyright (C) 2023 Intel Corp.
+# 
+# SPDX-License-Identifier: BSD-3-Clause
+
+deviceList:
+  - name: pos-mqtt
+    profileName: pos
+    description: MQTT device for POS
+    labels:
+      - MQTT
+      - pos
+    protocols:
+      mqtt:
+        CommandTopic: EdgeXPosCommand
```

Comment:
Should all others start with caps as this one? I see in the toml file that all started with caps.

Reply:
Toml case didn't matter. Yaml is camelCase except for the Protocol map keys which are up to the Device Service. In this case they use lowe case for the protocol name `mqtt` and PascalCase for the protocol setting name `CommandTopic`